src/collections/_documentation/platforms/dotnet/nlog.md

```diff
@@ -0,0 +1,169 @@
+    .AddSentry(o =>
+    {
+        // The NLog integration will initialize the SDK if DSN is set:
+        o.Dsn = new Dsn("___PUBLIC_DSN___"));
+    });  
+```
+
+> NOTE:
+The SDK only needs to be initialized once. If a `DSN` is made available to this integration, by default it **will** initialize the SDK. If you do not wish to initialize the SDK via this integration, set the `InitializeSdk` flag to **false**. Not providing a DSN or leaving it as `null` instructs the integration not to initialize the SDK and unless another integration initializes it or you call `SentrySdk.Init`, the SDK will stay disabled.
+
+The SDK can also be configured via `NLog.config` XML file:
+
+```xml
+<?xml version="1.0" encoding="utf-8" ?>
+<nlog xmlns="http://www.nlog-project.org/schemas/NLog.xsd"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+>
+  <extensions>
+    <add assembly="Sentry.NLog" />
+  </extensions>
+
+  <targets>
+    <target xsi:type="Sentry" name="sentry"
+            dsn="___PUBLIC_DSN___"
+            layout="${message}"
+            debug="true"
+            breadcrumbLayout="${message}"
+            minimumBreadcrumbLevel="Debug"
+            minimumEventLevel="Error">
+
+      <!-- Advanced options can be configured here-->
+      <options
+          environment="Development"
+          attachStacktrace="true"
+          sendDefaultPii="true"
+          shutdownTimeoutSeconds="5"
+        >
+        <!--Advanced options can be specified as attributes or elements-->
+        <includeEventDataOnBreadcrumbs>true</includeEventDataOnBreadcrumbs>
+      </options>
+
+      <!--Add any desired additional tags that will be sent with every message -->
+      <tag name="logger" layout="${logger}" />
+      <tag name="example" layout="sentry-nlog" />
+    </target>
+  </targets>
+
+  <rules>
+    <logger name="*" writeTo="sentry" />
+  </rules>
+</nlog>
+```
+
+### Options
+
+#### MinimumBreadcrumbLevel
+
+A `LogLevel` which indicates the minimum level a log message has to be included as a breadcrumb. By default this value is `Info`.
+
+#### MinimumEventLevel
+
+A `LogLevel` which indicates the minimum level a log message has to be sent to Sentry as an event. By default this value is `Error`.
+
+#### InitializeSdk
+
+Whether or not this integration should initialize the SDK. If you intend to call `SentrySdk.Init` yourself you should set this flag to `false`.
+
+#### IgnoreEventsWithNoException
+
+To ignore log messages that don't contain an exception.
+
+#### SendEventPropertiesAsData
+
+Determines whether event-level properties will be sent to sentry as additional data. Defaults to true.
+
+#### SendEventPropertiesAsTags
+
+Determines whether event properties will be sent to sentry as Tags or not. Defaults to false.
+
+#### IncludeEventDataOnBreadcrumbs
+
+Determines whether or not to include event-level data as data in breadcrumbs for future errors. Defaults to false.
+
+#### BreadcrumbLayout
+
+Custom layout for breadcrumbs.
```

Comment:
I'd been assuming anyone who is planning on using this would be familiar with `NLog` layouts, but it might be good to put a link to the documentation for layouts, e.g. either https://nlog-project.org/config/?tab=layout-renderers or https://github.com/NLog/NLog/wiki/Layout-Renderers, just to make it easy if someone is coming from sentry to nlog, for example, instead of the other way around.

Might not be that big a deal, but it could come in handy for people new to NLog.